### PR TITLE
Feature/gelu shift negative

### PR DESCRIPTION
**Pull request by yardeny-sony**
## Pull Request Description:
Add GELU activation operator to shift negative substitustions in both keras and pytorch.

## Checklist before requesting a review:
- [x] I set the appropriate labels on the pull request.
- [ ] I have added/updated the release note draft (if necessary).
- [x] I have updated the documentation to reflect my changes (if necessary).
- [x] All function and files are well documented. 
- [x] All function and classes have type hints.
- [x] There is a licenses in all file.
- [x] The function and variable names are informative. 
- [x] I have checked for code duplications.
- [x] I have added new unittest (if necessary).

### CODE REVIEW

**Review by Idan-BenAmi — APPROVED** [review]

**Comment by ofirgo** [comment]
@yardeny-sony, please add a description to the PR, explaining the main changes and possible design choices that were made (if there are such)
